VERSION-RANGE-SPEC.rst

```diff
@@ -652,6 +652,8 @@ These are generic schemes, to use sparingly for special cases:
 - **all**: a generic versioning scheme for a range containing all versions.
   ``vers:all/*`` is the only valid vers form for this scheme.
 
+- **intdot**: a generic versioning scheme that allows version components to be specified as integers separated by dots, e.g. ``10.234.5.12``. Versions specified in this scheme consist of ASCII digits only, formatted with only non-negative integers, and ignoring leading zeros. Interpretation of the version should stop at the first character that is not a digit or a dot.
```

Comment:
is this really a new generic schema? looks like a subset of the  already defined `generic` to me.

Reply:
This is fine as all, it is in a separate section.

Reply:
I think the overall idea is to get rid of "generic" and add more precise schemes instead, like intdot and [lexicographic](https://github.com/package-url/purl-spec/issues/265). 
I would suggest that in a separate PR I add the lexicographic scheme and remove the "generic" point.

Reply:
understood. then LGTM